integration/integration_test.go

```diff
@@ -177,9 +178,39 @@ func readSpdxJson(t *testing.T, filePath string) *spdx.Document2_2 {
 		return bom.Relationships[i].RefB.ElementRefID < bom.Relationships[j].RefB.ElementRefID
 	})
 
+	if bom.CreationInfo != nil {
+		sort.Slice(bom.CreationInfo.Creators, func(i, j int) bool {
+			if bom.CreationInfo.Creators[i].Creator != bom.CreationInfo.Creators[j].Creator {
+				return bom.CreationInfo.Creators[i].Creator < bom.CreationInfo.Creators[j].Creator
+			}
+			return bom.CreationInfo.Creators[i].CreatorType < bom.CreationInfo.Creators[j].CreatorType
+		})
+	}
```

Comment:
`Creators` is a slice. Do we need to sort it?

Reply:
I've done it to be on the safe side, so the goldan file will not need to be changed to the exact order.
I can remove it if you like.

Reply:
IMO, the golden file should be sorted in the exact order. We can remove it.
It would be great if `bom.CreationInfo.Creators` implements [sort.Interface](https://pkg.go.dev/sort#Interface), though.